x/claims/types/claims_record.go

```diff
@@ -3,12 +3,17 @@ package types
 import (
 	"errors"
 	"fmt"
+	gomath "math"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// IBCTriggerAmt is the amount required to trigger a merge/migration of claims records
-var IBCTriggerAmt = "63743"
+var (
+	// IBCTriggerAmt is the amount required to trigger a merge/migration of claims records
+	IBCTriggerAmt = "63743"
+	// GenesisDust is the amount of aevmos sent on genesis for accounts to be able to claim
+	GenesisDust = int64(gomath.Pow10(15))
```

Comment:
nit, I think this should be a string instead (same as `IBCTriggerAmt`) for consistency

Reply:
Thats a good point. Although I decided to do it like this for precission given the amount of 0 this numbers has. Happy to change it if you still think its worth it though

Reply:
Let's change it on `main`